R/scripts/create_seeding_nonUS.R

```diff
@@ -0,0 +1,120 @@
+##
```

Comment:
What is this file doing that create_seeding.R isn't doing?

Reply:
its old, meant to delete. Deleted.